config/vufind/EDS.ini

```diff
@@ -403,3 +403,18 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
```

Comment:
I'm glad that the functionality is looking good now (thanks for re-testing, @maccabeelevine!). Before we merge, though, I think it might make sense to adjust the configuration a bit to make it more future-proof for the intended next step. I'd suggest that perhaps we could revise the configs to something like this:

```
; This section controls how the authors of EDS records are displayed.
[AuthorDisplay]
; Control which data is used to display author information in search results.
; - 'Short' (the default) is fetched from the RecordInfo section of a record (the 
;   'BibRecord/BibRelationships/HasContributorRelationships/*PersonEntity/Name/NameFull' data).
;   Also, when you set 'ResultListPage' to 'Short' you can change the default delimiter "; " between
;   the author names by overriding the styles defined in the EDS.css file
;   under: 'Author name delimiters'.
; - 'Long' is fetched from the item 'Au' group data.
ResultListFormat = 'Short'
; This setting controls how many author names to display when ResultListFormat is
; set to 'Short'. It is ignored if the setting is 'Long'. If a record has more author names
; than you define in AuthorNameLimit, then VuFind will display the more_authors_abbrev text.
;AuthorNameLimit = 3
```
...and then also of course adjust the code to use the new config setting names.

Note that this is just an initial proposal, and there might be other/better ways to adjust the config. The idea here is that we have location-specific "format" settings (so ResultListFormat now, and in the future, RecordFormat as well -- and maybe a DefaultFormat as fallback) but we have only one AuthorNameLimit. The AuthorNameLimit is applied to "Short" but not "Long" which makes logical sense (the "Short" list is truncated). But that raises a new question: do we need separate settings for data source vs. length? i.e. might you want a "Short" (truncated) list that's sourced from the "Long" ('Au' group) data? I'm not really sure -- but if that kind of scenario is conceivable, maybe we really want separate DataSource and Format settings, to separate the data source from the display format. Or maybe we want the AuthorNameLimit to be differentiated by location in parallel with the Format setting, instead of being tied to "Short" format.

What do you think? I don't think it will be too hard to implement any of these proposed ideas, but I think it's a good idea to agree on the most useful/logical configuration format before we merge this, so the follow-up work will be simple and straightforward to implement, and we won't end up changing things that have already been committed.

Reply:
The format of the authors in Au can differ significantly depending on the content provider, therefore I don't think we should try to apply the limit in that case. Maybe ShortAuthorLimit instead of AuthorNameLimit?

Reply:
thanks @demiankatz and @berndoberknapp !

Sorry for my delay providing a followup.

What I've added in my latest follow-up is:

1. Added section name of [AuthorDisplay] as you suggested @demiankatz  - and made the code use it.
2. Added config name ResultListFormat as you suggested @demiankatz - and made the code use it.
3. Added the config name ShortAuthorLimit, as you suggested @berndoberknapp  - and made the code use it.

--
I think what we could do in the follow-up commit is:
- Add a DetailPageFormat config if that is also set to 'Short' then VuFind looks at the value of  the single ShortAuthorLimit config to decide how many author names to display.

i.e. we could have a config like so:
```
[AuthorDisplay] 
ResultListFormat = 'Short'
DetailPageFormat = 'Short'
ShortAuthorLimit = 3
```

Could you please let me know your thoughts on this idea?